.github/workflows/build.yml

```diff
@@ -97,7 +73,7 @@ jobs:
       env:
         TOX_SPELL_REQUIRE: both
       run: |
-        python -m tox
```

Comment:
I do not wish to sequence it in this way. I'd like each test to run and fail independently.

Reply:
Each will run separately and fail independently.  That is why we use a matrix.

Reply:
I originally had it done this way so that I could independently control `continue-on-error`. Sometimes I may attempt to add support for a future version, but dependencies or Python versions can be buggy in early stages (beta/alpha).